x-team/auto-ui
# Add required field indicator to input fields

**Release Type:** *New feature* 


Fixes x-team/auto#1356

## Description

- Adds a required field indicator to input fields.

## Checklist



## Related PRs

branch | PR
------ | ------
`1356-show-a-required-field-indictor-on-required-fields` | [Auto](https://github.com/x-team/auto/pull/1371)

## Steps to Test or Reproduce

- Navigate to the `Text` component section.
  - Make sure the `required` prop works.
- Navigate to the `InputField` component section.
  - Make sure the `required` prop works.

## Screenshots

\# 1 - `Text` component

![screen shot 2018-07-24 at 09 05 47](https://user-images.githubusercontent.com/8301142/43139939-0868e910-8f21-11e8-8b21-134df02c6063.png)

![screen shot 2018-07-24 at 09 06 42](https://user-images.githubusercontent.com/8301142/43139947-0bca91b2-8f21-11e8-8c8c-832cc1f57186.png)

\# 2 - `InputField` component

![screen shot 2018-07-23 at 21 40 06](https://user-images.githubusercontent.com/8301142/43111865-2948a978-8ec1-11e8-8f13-de67907d993e.png)
